lib/pages/chat/input_bar.dart

```diff
@@ -516,6 +522,11 @@ class InputBar extends StatelessWidget {
                 // fix for the library for now
                 // it sets the types for the callback incorrectly
                 onChanged!(text);
+                // #Pangea
+                if (maxLength != (controller?.text.length == 1000)) {
```

Comment:
Please move this boolean logic to the function ```refreshOnChange```.
It's good practice to have as little state logic in the view code as possible, and to have that logic in the controller/state instead.